googletest/cmake/internal_utils.cmake

```diff
@@ -159,7 +160,7 @@ function(cxx_library_with_type name type cxx_flags)
       PROPERTIES
       COMPILE_DEFINITIONS "GTEST_CREATE_SHARED_LIBRARY=1")
   endif()
-  if (CMAKE_USE_PTHREADS_INIT)
+  if (NOT gtest_disable_pthreads AND NOT MINGW AND CMAKE_USE_PTHREADS_INIT)
```

Comment:
As this term is repeated identically to line 167 in `CMakeLists.txt`, I personally would prefer to set one CMake variable with this term, so future changes are in one place only. As you suggested already in the issue's discussion.

Reply:
But I'm not a maintainer here :-)

Reply:
Sure, that i can do easily :)

Reply:
+1 on @hethi 's suggestion

Reply:
Already done.